orchestration/dags/data_gcp_dbt/models/intermediate/applicative/int_applicative__collective_offer.sql

```diff
@@ -56,7 +56,8 @@ with collective_stocks_grouped_by_collective_offers as (
         co.national_program_id,
         NULL as collective_offer_template_beginning_date,
         NULL as collective_offer_template_ending_date,
-        case when cs.collective_stock_is_bookable and co.collective_offer_is_active then TRUE else FALSE end as collective_offer_is_bookable,
+        case when cs.collective_stock_is_bookable and co.collective_offer_is_active
```

Comment:
cs.collective_stock_is_bookable and co.collective_offer_is_active and collective_offer_validation = "APPROVED" as collective_offer_is_bookable

plutot qu'un case when ?